src/main/include/Constants.h

```diff
@@ -11,6 +12,8 @@ constexpr double NEO550_RES{ 42 };
 constexpr double INTAKE_GEAR_RATIO{203.636364};
 constexpr double ARM_ENC_CONVERSION_FACTOR{360.0 / (NEO550_RES * INTAKE_GEAR_RATIO)};
 
+const std::string LL_TABLE_NAME{ "" };
```

Comment:
May I be a stickler about spelling things out?

```suggestion
const std::string LIMELIGHT_TABLE_NAME{ "" };
```